index.html

```diff
@@ -44,9 +44,10 @@
           <!-- this is our audio div.  It holds the audio file. -->
           <audio id="media" controls="controls">
             <!-- Make sure this points to the actual mp3 file you are analyzing -->
-            <source src="https://storycorpsorg-staging.s3.amazonaws.com/uploads/walsh2-1.mp3"
+            <source src="http://2017.hackinghistory.ca/audio/uden-bhutia-ecotourism.mp3"
                     type="audio/mp3" />
           </audio>
+          <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/3/3d/Kanchendzonga.jpg/1280px-Kanchendzonga.jpg#left"><img id="pic" src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/3d/Kanchendzonga.jpg/1280px-Kanchendzonga.jpg"/>
```

Comment:
the `a` tag is not self-=closing, so you have an open tag that goes from here to the end of the web page. Can you fix?